.jenkinsci-new/artifacts.groovy

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env groovy
+
+def uploadArtifacts(filePaths, uploadPath, artifactServers=['nexus.iroha.tech']) {
+  def filePathsConverted = []
+  agentType = sh(script: 'uname', returnStdout: true).trim()
+  filePaths.each {
+    fp = sh(script: "ls -d ${it} | tr '\n' ','", returnStdout: true).trim()
+    filePathsConverted.addAll(fp.split(','))
+  }
+  def shaSumBinary = 'sha256sum'
+  def md5SumBinary = 'md5sum'
+  def gpgKeyBinary = 'gpg --armor --detach-sign --no-tty --batch --yes --passphrase-fd 0'
+  if (agentType == 'Darwin') {
+    shaSumBinary = 'shasum -a 256'
+    md5SumBinary = 'md5 -r'
+    gpgKeyBinary = 'GPG_TTY=\$(tty) gpg --pinentry-mode loopback --armor --detach-sign --no-tty --batch --yes --passphrase-fd 0'
+  }
+  sh "> \$(pwd)/batch.txt"
+
+  withCredentials([file(credentialsId: 'ci_gpg_privkey', variable: 'CI_GPG_PRIVKEY'), string(credentialsId: 'ci_gpg_masterkey', variable: 'CI_GPG_MASTERKEY')]) {
+    if (!agentType.contains('MSYS_NT')) {
+      sh "gpg --yes --batch --no-tty --import ${CI_GPG_PRIVKEY} || true"
+    }
+    filePathsConverted.each {
+      sh "echo ${it} >> \$(pwd)/batch.txt;"
+      sh "$shaSumBinary ${it} | cut -d' ' -f1 > \$(pwd)/\$(basename ${it}).sha256"
+      sh "$md5SumBinary ${it} | cut -d' ' -f1 > \$(pwd)/\$(basename ${it}).md5"
+      // TODO @bakhtin 30.05.18 IR-1384. Make gpg command options and paths compatible with Windows OS.
```

Comment:
@bakhtin is this something that we still need to do?

Reply:
I would rather not focus much for Windows support

Reply:
let's delete it please... @BulatSaif

Reply:
done